2023-03-23 14:39:34 [ERROR] __main__: PlateRecording failed: 'NoneType' object is not subscriptable
13
Traceback (most recent call last):
12
  File "/app/main.py", line 160, in process
11
    recordings = _load_from_dir(tmpdir, plate_recording_args)
10
  File "/app/main.py", line 38, in _load_from_dir
9
    recordings = list(PlateRecording.from_directory(recording_dir, **plate_recording_args))
8
  File "/app/venv/lib/python3.9/site-packages/pulse3D/plate_recording.py", line 631, in from_directory
7
    yield PlateRecording(of, **kwargs)
6
  File "/app/venv/lib/python3.9/site-packages/pulse3D/plate_recording.py", line 380, in __init__
5
    self.platemap_name = self.wells[0][PLATEMAP_NAME_UUID]
4
TypeError: 'NoneType' object is not subscriptable